cluster-autoscaler/cloudprovider/builder/builder_all.go

```diff
@@ -0,0 +1,58 @@
+// +build !gce,!aws,!azure,!kubemark
```

Comment:
I would prefer having explicit tag for that `all_cp` so we do not have to update this list when new cloud provider is added.
Given that adding `_cp` suffix to cloud-provider specific tags seems reasonable to me.